files/src/main/scala/io/iteratee/files/FileModule.scala

```diff
@@ -18,37 +18,45 @@ import scala.collection.JavaConverters._
 import scala.util.{ Left, Right }
 
 trait FileModule[F[_]] { this: Module[F] { type M[f[_]] <: MonadError[f, Throwable] } =>
-  protected def captureEffect[A](a: => A): F[A]
+  protected def captureEffect[A](a: => A): F[A] = F.catchNonFatal(a)
```

Comment:
So, as I have discussed on the thread here:

https://github.com/typelevel/cats/issues/1316

I don't see why `F.catchNonFatal(a)` possibly coupled with `Eval.always` is not what we want in practice.

I feel like making a typeclass creates a veneer of rigor that is actually absent, and I'd frankly rather just stick to the mechanical style of Eval + MonadError, or Eval + Monad if it can't fail (but very few true effects can't fail).

If we have a typeclass hear, we can certainly generate those from something in cats, but that feels more hypothetical to me.

Given that this only adds `liftMEval` which I think is useful to have for laziness if nothing else, and it leaves the existing modules as is, it seems a conservative approach.